src/libfuncs/felt252_dict.rs

```diff
@@ -57,7 +56,7 @@ pub fn build_new<'ctx, 'this>(
     let op = runtime_bindings.dict_alloc_new(context, helper, entry, location)?;
     let dict_ptr = op.result(0)?.into();
 
-    entry.append_operation(helper.br(0, &[segment_arena, dict_ptr], location));
+    entry.append_op_result(helper.br(0, &[segment_arena, dict_ptr], location));
```

Comment:
dont use append_op_result if the operation has no result, it was correct before the change